src/main/kotlin/blackjack/domain/Hand.kt

```diff
@@ -0,0 +1,24 @@
+package blackjack.domain
+
+class Hand(cardList: List<Card> = emptyList()) {
```

Comment:
```suggestion
class Hand() {
```

Hand에 생성자로 cardList는 없어도 될 것 같네요.

Reply:
Hand 생성자의 cardList를 다음과 같이 사용중이었습니다..!
```
Player(
    name = it,
    Hand(
        cards = deck.popMany(count = BlackjackRule.INITIAL_CARD)
    )
)
```